netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java

```diff
@@ -121,7 +122,8 @@ public class NettyClientHandlerTest extends NettyHandlerTestBase<NettyClientHand
   private NettyClientStream.TransportState streamTransportState;
   private Http2Headers grpcHeaders;
   private long nanoTime; // backs a ticker, for testing ping round-trip time measurement
-  private int maxHeaderListSize = Integer.MAX_VALUE;
+  private int maxHeaderListSize = 8000;
```

Comment:
Don't change these values for all tests. Instead, set them at the beginning of your test and call `manualSetUp();`.

Reply:
Done.